internal/auth-service/auth-service-config.go

```diff
@@ -0,0 +1,34 @@
+package auth_service
+
+import (
+	configv1alpha1 "github.com/liqotech/liqo/apis/config/v1alpha1"
+	"github.com/liqotech/liqo/pkg/clusterConfig"
+)
+
+func (authService *AuthServiceCtrl) GetAuthServiceConfig(kubeconfigPath string) error {
+	waitFirst := make(chan bool)
```

Comment:
Since we are using this chan only once we could use a `chan struct{}`. In order to Communicate that the event has happened, you can close it. In this case, line 19 wouldn't be needed anymore.